cypress/support/pages/tasks/pageCommonTasks.js

```diff
@@ -15,5 +22,88 @@ class Tasks_Common {
     cy.get(Tasks_Common.#taskForm).find(Tasks_Common.#submitButton).click()
     //cy.get(Tasks_Common.#submitButton).should('contain', '').click()
   }
+
+  clickOnAddTask() {
+    cy.get(Tasks_Common.#addTaskButton).click()
+  }
+
+  seeCreateTaskPage (taskTitle) {
+    if (taskTitle === '-'){
+      cy.get(Tasks_Common.#taskForm)
+        .get(Tasks_Common.#taskNameInput)
+        .should('be.empty')
+    } else {
+      cy.get(Tasks_Common.#taskForm)
+        .get(Tasks_Common.#taskNameInput)
+        .should('have.value', taskTitle)
+    }
+  }
+
+  enterTaskTitle (taskTitle) {
+    cy.get(Tasks_Common.#taskNameInput).clear()
+    cy.get(Tasks_Common.#taskNameInput).type(taskTitle)
+  }
+
+  setTaskText (taskText) {
+    cy.get(Tasks_Common.#homeWorkDescriptionP)
```

Comment:
Please see above comment for the selector

Reply:
changed it to #homeworkDescription